entity.go

```diff
@@ -57,3 +55,13 @@ func (e *Entity) SetPosition(x, y int) {
 func (e *Entity) SetCell(x, y int, c *Cell) {
 	renderCell(&e.canvas[x][y], c)
 }
+
+// Fill fills the canvas of the Entity with
+// a c Cell.
```

Comment:
:+1: 
Actually, if you want to rephrase the whole func description, just go ahead and put it here - I'll never be a native english speaker :smile: